test/models/vote_test.rb

```diff
@@ -0,0 +1,59 @@
+require "test_helper"
+
+describe Vote do
+
+  before do
+    @arbitrary_vote = Vote.new(user: User.find_by(username: "first-name123"), work: Work.find_by(category: 'movie'))
+    @valid_unique_vote = votes(:different_one)
+    # Vote.first
+  end
+
+  describe 'validations' do
+    it 'is valid when a valid user and valid vote are present' do
+
+      empty_hash = {}
+      expect(@valid_unique_vote.valid?).must_equal true
+      expect(@valid_unique_vote).must_be_kind_of Vote
+      expect(@valid_unique_vote.errors.messages).must_equal empty_hash
+    end
+
+    it 'is is invalid if user and work uninstantiated' do
+
+      @arbitrary_vote.user = nil
+      @arbitrary_vote.work = nil
+
+      expect(@arbitrary_vote.valid?).must_equal false
+      expect(@arbitrary_vote.errors.messages).must_include :user, :work
+    end
+
+    it 'is is invalid if user id and work id are nil' do
```

Comment:
how is this test different than the one above? If you had tests on relations, I would even argue this is even more redundant